rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] :book and :person should be getter and setter, kindly include them on attr_accessor